### PR TITLE
Add basic skeleton for Home & Posts with ind Post

### PR DESCRIPTION
**Pull request by a-sh-dev**
- add posts
- add post
- and update home page to display posts
- all basic skeleton for view only

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import '@testing-library/jest-dom';
 
 // If we're using ENZYME - https://enzymejs.github.io/enzyme/
-// import Enzyme from 'enzyme';
-// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+import Enzyme from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```
Comment (NatachaBartu) [inline src/setupTests.js:6]: nice you add the adapter enzyme! nice work!

**Comment by thangzathang** [comment]
Most code is new and you only deleted a few imports so it should be fine. Nice work.

File: src/views/home/Home.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
       <div>
         <h1>Home Page</h1>
+        <h3>Searchbar</h3>
+        <h3>Featured Posts</h3>
+        <main>
+          <Posts />
+        </main>
       </div>
     </>
   );
```
Comment (NatachaBartu) [inline src/views/home/Home.js:16]: Good Start structure Angie!

**Review by NatachaBartu — COMMENTED** [review]
Well done Angie! Good structure for Post

File: src/views/home/Home.js (hunk 2 above)
Comment (a-sh-dev) [reply src/views/home/Home.js:16]: thanks team, will merge now, it's very basic for now..

**Pull request merged**